bigtable/integration_test.go

```diff
@@ -1400,8 +1401,10 @@ func TestIntegration_BackupIAM(t *testing.T) {
 	if err := adminClient.CreateTable(ctx, table); err != nil {
 		t.Fatalf("Creating table: %v", err)
 	}
+
 	// Create backup.
-	backup := "backup"
+	backupUUID := uuid.New()
```

Comment:
Other manuals make use of https://github.com/googleapis/google-cloud-go/blob/main/internal/uid/uid.go#L54. You can avoid a dependency if you take this approach too 😄 .

Reply:
Good call! Done.